swedish.ini

```
@@ -0,0 +1,666 @@
+Left_Tip = "Ändra Tangenten för Vänster"
+Right_Tip = "Ändra Tangenten för Höger"
+Jump_Tip = "Ändra Tangenten för Hopp"
+Fire_Tip = "Ändra Tangenten för Skjut"
+DiagonalAim_Tip = "Ändra Tangenten för att Sikta Diagonalt"
+AimUp_Tip = "Ändra Tangenten för att Sikta Uppåt"
+AimDown_Tip = "Ändra Tangenten för att Sikta Nedåt"
+AimLock_Tip = "Ändra Tangenten för att Låsa Siktet"
+Morph_Tip = "Ändra Tangenten för att aktivera Morphning"
+ArmWeapon_Tip = "Ändra Tangenten för att aktivera Missiler"
+WeaponSelect_Tip = "Ändra Tangenten för att Växla Vapen"
+WeaponCancel_Tip = "Ändra Tangenten för att Nollställa Vapen"
+Start_Tip = "Ändra Tangenten för Start / Meny"
+MenuAccept_Tip = "Ändra Tangenten för att Välja i menyer"
+MenuBack_Tip = "Ändra Tangenten för att gå Bakåt till föregående meny"
+
+[OptionsControlJoypad]
+Title_Joypad = "Handkontrollsinställningar"
+Title_XBox360Joypad = "XBox 360-Handkontroll"
+Vibration = "Vibration"
+Vibration_Tip = "Handkontrollen Vibrerar vid Händelser"
+Deadzone = "Deadzone"
+Deadzone_Tip = "Ändra den analoga Dödzonen"
+AnalogWalking = "Analog Gång"
+AnalogWalking_Tip = "Aktivera för att gå genom att använda den analoga styrspaken"
+XAxis = "X-Axel"
+XAxis_Tip = "Välj riktning på den Horisontella Axeln"
+YAxis = "Y-Axel"
+YAxis_Tip = "Välj riktning på den Vertikala Axeln"
+Up_Tip = "Ändra Knappen för Upp"
+Down_Tip = "Ändra Knappen för Ned"
+Left_Tip = "Ändra Knappen för Vänster"
+Right_Tip = "Ändra Knappen för Höger"
+Jump_Tip = "Ändra Knappen för Hopp"
+Fire_Tip = "Ändra Knappen för Skjut"
+DiagonalAim_Tip = "Ändra Knappen för att Sikta Diagonalt"
+AimUp_Tip = "Ändra Knappen för att Sikta Uppåt"
+AimDown_Tip = "Ändra Knappen för att Sikta Nedåt"
+AimLock_Tip = "Ändra Knappen för att Låsa Siktet"
+Morph_Tip = "Ändra Knappen för Morphning"
+ArmWeapon_Tip = "Ändra Knappen för att Aktivera Missiler"
+WeaponSelect_Tip = "Ändra Knappen för att Växla Vapen"
+WeaponCancel_Tip = "Ändra Knappen för att Nollställa Vapen"
+Start_Tip = "Ändra Knappen för Start / Meny"
+MenuAccept_Tip = "Ändra Knappen för att Välja i menyer"
+MenuBack_Tip = "Ändra Knappen för att gå Bakåt till föregående meny"
+
+[PauseMenu]
+Resume = "Fortsätt Spela"
+Resume_Tip = "Fortsätt spela det nuvarande spelet"
+Restart = "Omstart från senaste Spartillfälle"
+Restart_Tip = "Avbryt ditt nuvarande spel och ladda det senast sparade spelet"
+Options = "Inställningar"
+Options_Tip = "Ändra spelinställningarna"
+Quit = "Avsluta Spel"
+Quit_Tip = "Avbryt ditt nuvarande spel och återgå till huvudmenyn"
+
+[QuitMenu]
+Exit = "Avsluta - Är du säker?"
+Reload = "Omstart - Är du säker?"
+Warning = "VARNING: Alla osparade handlingar kommer förloras"
+
+[Subscreen]
+Title_Map = "PLANET SR-388"
+Title_Options = "Pausmeny"
+Menu_Tip = "| -> Meny"
+Marker_Tip = "| - Placera Markör"
+MarkerPos_Tip = "| - Sätt Position av Markör"
+DeleteMarker_Tip = "| - Radera Markör"
+EditMarker_Tip = "| - Ändra Markör"
+SelectItem_Tip = " - Välj artikel"

```

"Artikel" has a different meaning than what's intended here, "Föremål" better fits the use case.
```suggestion
SelectItem_Tip = " - Välj föremål"

```